modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/code_mirror_keyboard_message/CodeMirrorKeyboardMessage.tsx

```diff
@@ -15,8 +16,25 @@ interface IProps {
 export default function CodeMirrorKeyboardMessage({
 	keyIsEnabled = false,
 }: IProps) {
+	const REDUCE_TIMEOUT_MS = 4000;
```

Comment:
I'd move this outside the component since it's not something that will change with each render.

```
const REDUCE_TIMEOUT_MS = 4000;

export default function CodeMirrorKeyboardMessage({
	keyIsEnabled = false,
}: IProps) {
    const [reduce, setReduce] = useState<boolean>(false);
...
```

Reply:
Rebased, changed and pushed the change, thanks @thektan

Reply:
I'm also keeping the className Remove since it is not used